### PR TITLE
Randomized resource names to avoid 409 errors

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Fixing error:

googleapi: Error 409: The resource 'entity.managedZone' named 'peering-zone' already exists, alreadyExists

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_whitespace
 Failed whitespace check. More details below. 
 ```bash 
Checking for trailing whitespace
./dns_managed_zone_service_directory/main.tf:21: 
Error: Trailing whitespace found in the lines above.
Checking for missing newline at end of file
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
dns_managed_zone_service_directory/main.tf
--- old/dns_managed_zone_service_directory/main.tf
+++ new/dns_managed_zone_service_directory/main.tf
@@ -18,7 +18,7 @@
 resource "random_id" "zone_prefix" {
   byte_length = 8
 }
- 
+
 resource "google_dns_managed_zone" "sd_zone" {
   provider = google-beta
 
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
dns_managed_zone_service_directory/main.tf
--- old/dns_managed_zone_service_directory/main.tf
+++ new/dns_managed_zone_service_directory/main.tf
@@ -20,7 +20,7 @@
 }
 
 resource "google_dns_managed_zone" "sd_zone" {
-  provider = google-beta
+  provider    = google-beta
   name        = "peering-zone-${random_id.zone_prefix.hex}"
   dns_name    = "services.example.com."
   description = "Example private DNS Service Directory zone"
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

File: dns_managed_zone_private_peering/main.tf [hunk 1]
```diff
@@ -15,8 +15,12 @@
  */
 
 # [START dns_managed_zone_private_peering]
+resource "random_id" "zone_prefix" {
```
Comment (rogerthatdev) [inline dns_managed_zone_private_peering/main.tf:18]: value for `zone_prefix` is used as a suffix at end of zone name.  let's rename this resource to `zone_suffix`

File: dns_managed_zone_service_directory/main.tf [hunk 2]
```diff
@@ -15,10 +15,13 @@
  */
 
 # [START dns_managed_zone_service_directory]
-resource "google_dns_managed_zone" "sd_zone" {
-  provider = google-beta
+resource "random_id" "zone_prefix" {
```
Comment (rogerthatdev) [inline dns_managed_zone_service_directory/main.tf:18]: value for zone_prefix is used as a suffix at end of zone name. let's rename this resource to zone_suffix

**Review by rogerthatdev — COMMENTED** [review]
single nit on prefix resource naming

**Review by rogerthatdev — APPROVED** [review]

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_headers
 All files need a license header. Please make sure all your files include the appropriate header. A helper tool available [here](https://github.com/google/addlicense). 
 ```bash 
Checking file headers
./compute_add_persistent_disk/main.tf
 ```

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request merged**